rust/Cargo.toml

```
@@ -66,14 +66,19 @@ version = "0.9.13"
 features = [ "parallel" ]
 version = "0.9.13"
 
+[dependencies.snarkvm]
+features = [ "cli" ]
+version = "0.9.13"
```

```suggestion
workspace = true
```
nit, but can we make this a workspace dep?
note: editing the above line may not compile out of the box